AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -34,6 +29,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)removeDelegate:(id<MSAuthTokenContextDelegate>)delegate;
 
+/**
+ * Clears cached token and account id.
+ */
+- (void)clearAuthToken;
+
+/**
+ * Sets current auth token and account id.
+ */
+- (void)setAuthToken:(NSString *)authToken withAccountId:(NSString *)accountId;
+
+/**
+ * Retrieves auth token.
+ */
+- (nullable NSString *)getAuthToken;
```

Comment:
method name should be `authToken:` instead of `getAuthToken` per Objective C convention.